packages/client/hmi-client/src/workflow/tera-workflow.vue

```diff
@@ -228,13 +228,15 @@
 			:title="currentActiveNode.displayName"
 			:tooltip="'A brief description of the operator.'"
 		>
-			<component
-				:is="drilldownRegistry.get(currentActiveNode.operationType)"
-				:node="currentActiveNode"
-				@append-output-port="(event: any) => appendOutputPort(currentActiveNode, event)"
-				@update-state="(event: any) => updateWorkflowNodeState(currentActiveNode, event)"
-			>
-			</component>
+			<section tabName="Wizard">
+				<component
+					:is="drilldownRegistry.get(currentActiveNode.operationType)"
+					:node="currentActiveNode"
+					@append-output-port="(event: any) => appendOutputPort(currentActiveNode, event)"
+					@update-state="(event: any) => updateWorkflowNodeState(currentActiveNode, event)"
+				>
+				</component>
+			</section>
```

Comment:
Just wrapping this in a tab called Wizard for the moment so that it does not interfere with devs when we merge.

Reply:
Perfect, this will be fixed with #2297